### ISSUE TITLE
[Tile] Card group keyboard interaction

### ISSUE THREAD

**Issue by thyhmdo** [carbon-design-system/carbon#18820]
Slack thread: https://ibm-studios.slack.com/archives/C0M053VPT/p1741699545046469

![Image](https://github.com/user-attachments/assets/9350c4ec-f910-4748-9573-a835b21fb688)

## Problem
There are multiple tab actions within a tile group, including interactive elements inside it.

## Focus Solution
- Define the keyboard interaction while ensuring that no interactive elements are placed inside an interactive card.
- If a card already has an assigned action when activated, we cannot use the same action to “open” it for navigating buttons and controls inside.
